### PR TITLE
mdcat 0.7.0 (new formula)

### PR DESCRIPTION
**Pull request by commitay**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/mdcat.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class Mdcat < Formula
+  desc "Show markdown documents on text terminals"
+  homepage "https://github.com/lunaryorn/mdcat"
+  url "https://github.com/lunaryorn/mdcat/archive/mdcat-0.6.0.tar.gz"
+  sha256 "bcc05615f6aeea1a406a481502abb12add62ac12c7f140bfe4430cda78039bb3"
+
+  depends_on "cmake" => :build
```
Comment (fxcoudert) [inline Formula/mdcat.rb]: I don't see anything that would need `cmake` here… other than that, the formula looks good!

Comment (arnested) [reply Formula/mdcat.rb]: `cargo` requires `cmake` during its install.

**Review by fxcoudert — APPROVED** [review]

**Review by ilovezfs — DISMISSED** [review]
It would be good for the `test do` block to use and verify markdown in some way.

**Comment by arnested** [comment]
For the test block how about:
```rb
  test do
    assert_match "┄┄[0.1] – 2018-01-14", shell_output("#{bin}/mdcat #{prefix}/CHANGELOG.md")
  end
```
I was about to submit the same PR when I discovred this one - so I already had something.

Also adding `head "https://github.com/lunaryorn/mdcat.git"` would be nice.

**Comment by commitay** [comment]
Updated test.

> Also adding head "https://github.com/lunaryorn/mdcat.git" would be nice.

I'll let @ilovezfs decide on this.

**Comment by fxcoudert** [comment]
Thanks @commitay for the pull request!

**Pull request closed without merging**